xbmc/cores/VideoPlayer/Edl.cpp

```diff
@@ -98,8 +98,9 @@ bool CEdl::ReadEditDecisionLists(const std::string& strMovie, const float fFrame
    * Only check for edit decision lists if the movie is on the local hard drive, or accessed over a
    * network share.
    */
+  std::string strMovie = fileItem.GetDynPath();
```

Comment:
this breaks edl for i.e. strm files

Reply:
Why? Previously fileItem.GetDynPath() was passed to the function. Now, the fileItem directly is passed and GetDynPath() used for backwards compatibility. Did I miss something there?

But yes, It does change the way recordings are detected (directly on the file item instead of by using the URL). From my point of view a more stable way.

Reply:
yes, passing fileitem is much better.